manifests/install.pp

```diff
@@ -26,8 +26,14 @@
     } else {
       $provider = undef
     }
+
+    $manage_package = $::jmxtrans::package_version ? {
+      ''      => present,
```

Comment:
If this were left in, this wouldn't work because `undef != ''`, and `$::jmxtrans::package_version` will never be an empty string since its type is `String[1]`. In general, you shouldn't check a variable against `''`, that's usually a sign you should be using `undef`.